pkg/azurefile/controllerserver.go

```diff
@@ -77,6 +77,9 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 			fileShareName = v
 		case diskNameField:
 		case fsTypeField:
+			if v != "" && v != cifs {
```

Comment:
in the `CreateVolume` func, could use this logic:
```
isDiskMount := (fsType != "" && fsType != cifs)
if isDiskMount {
 ...
}
```

Reply:
done